Examples/Python/python/acts/examples/reconstruction.py

```diff
@@ -934,17 +936,26 @@ def customLogLevel(custom: acts.logging.Level = acts.logging.INFO):
                 "Using RootVertexPerformanceWriter with smeared particles is not necessarily supported. "
                 "Please get in touch with us"
             )
+        kwargs = {}
+        if trajectories is not None:
+            kwargs["inputTrajectories"] = "trajectories"
+        else:
+            kwargs["inputAssociatedTruthParticles"] = associatedParticles
```

Comment:
I wondered if this would be cleaner to pass in the `RootVertexPerformanceWriter` with `"trajectories" if trajectories is not None else ""`

Reply:
Changed.